src/test/resources/gradle/cucumber/multipleRequests.feature

```diff
@@ -3,6 +3,6 @@ Feature: Multiple Requests
 
   Scenario: Multiple requests
     Given The server is running on port "33333"
-    When I send method "GET" for "/simple_get" to host at the specified port 3 times
+    When I send method "GET" for "/simple_get" to host at the specified port 1 time
```

Comment:
Do we need this test?

Reply:
Maybe we could just break this up into multple steps instead of specifying how many times the same request is sent. Repeating the same request doesn't seem like a typical real world scenario.

Reply:
@damonkelley I will work on modifying these acceptance tests in the next iteration.

Reply:
Had some extra time to modify them and implement this suggestion!